CHANGELOG.md

```diff
@@ -5969,3 +5969,4 @@
 [@hirasawayuki]: https://github.com/hirasawayuki
 [@grosser]: https://github.com/grosser
 [@mttkay]: https://github.com/mttkay
+[@FnControlOption]: https://github.com/FnControlOption
```

Comment:
You shouldn't edit this manually - your handle will be auto-added by the changelog entry.